cypress/integration/hooks.spec.js

```diff
@@ -102,6 +107,26 @@ context('Hooks', () => {
           });
         });
       });
+      context('successful redeliver hook', () => {
+        beforeEach(() => {
+          cy.redeliverHook();
+          cy.get('[data-test=redeliver-hook-1]').as('redeliverHook');
+        });
+
+        it('should show successful toasty.', () => {
```

Comment:
nit-picking but remove 'toasty' and remove the period '.'
```suggestion
        it('should show alert', () => {
```